src/script/interpreter.cpp

```diff
@@ -1354,6 +1357,16 @@ static bool VerifyWitnessProgram(const CScriptWitness& witness, int witversion,
         return set_success(serror);
     }
 
+    // Policy limit for stack size and stack element size
+    if (flags & SCRIPT_VERIFY_DISCOURAGE_BIG_P2WSH) {
+        if (stack.size() > 100)
+            return set_error(serror, SCRIPT_ERR_DISCOURAGE_BIG_P2WSH);
+        for (unsigned int i = 0; i < stack.size(); i++) {
+            if (stack.at(i).size() > 80)
```

Comment:
Apart from the hashing operations, there is not even an opcode in the scripting language that can process them.